Ensure local file server shuts down properly

the local file server (port 8642) runs when the LocalStorage class is instantiated in local.storage.ts, and does not always shut down properly. This can cause interference with tests:

```
        1) storageprovider
              tests for LocalStorage
                should return valid object url in LocalStorage:
       
             AssertionError [ERR_ASSERTION]: false == true
             + expected - actual
       
             -false
             +true
             
             at Context.<anonymous> (tests/storageprovider/storageprovider.test.ts:118:16)
             at processTicksAndRejections (node:internal/process/task_queues:95:5)
       
       
       
       npm ERR! Lifecycle script `test` failed with error: 
       npm ERR! Error: command failed 
       npm ERR!   in workspace: @etherealengine/server-core@1.4.0 
```

--- a/packages/server-core/src/media/storageprovider/local.storage.ts
+++ b/packages/server-core/src/media/storageprovider/local.storage.ts
@@ -34,6 +34,7 @@ import { MULTIPART_CUTOFF_SIZE } from '@etherealengine/common/src/constants/File
 
 import { FileBrowserContentType } from '@etherealengine/engine/src/schemas/media/file-browser.schema'
 import { getState } from '@etherealengine/hyperflux'
+import { ChildProcess } from 'child_process'
 import logger from '../../ServerLogger'
 import { ServerMode, ServerState } from '../../ServerState'
 import config from '../../appconfig'
@@ -81,9 +82,30 @@ export class LocalStorage implements StorageProviderInterface {
     this._store = fsStore(this.PATH_PREFIX)
 
     if (getState(ServerState).serverMode === ServerMode.API && !config.testEnabled) {
-      require('child_process').spawn('npm', ['run', 'serve-local-files'], {
-        cwd: process.cwd(),
-        stdio: 'inherit'
+      const child: ChildProcess = require('child_process').spawn(
+        'npx',
+        [
+          'http-server',
+          `${this.PATH_PREFIX}`,
+          '--ssl',
+          '--cert',
+          `${config.server.certPath}`,
+          '--key',
+          `${config.server.keyPath}`,
+          '--port',
+          '8642',
+          '--cors=*',
+          '--brotli',
+          '--gzip'
+        ],
+        {
+          cwd: process.cwd(),
+          stdio: 'inherit',
+          detached: true
+        }
+      )
+      process.on('exit', async () => {
+        process.kill(-child.pid!, 'SIGINT')
       })
     }
     this.getOriginURLs().then((result) => (this.originURLs = result))

